src/scss/dp/overrides/components/_improve-this-page.scss

```diff
@@ -3,13 +3,48 @@
     background-color: $astral;
     color: $white;
     padding: 10px 15px 10px;
+    display: flex;
+    justify-content: space-between;
+
+    @include breakpoint(sm) {
+      flex-wrap: wrap;
+    }
+
+    h3 {
```

Comment:
just wondering if you don't need to use classes instead of overwriting h3? something like  `m-bottom--0` (sorry if that is incorrect as I am not familiar with this system just took it form https://github.com/ONSdigital/dp-design-system/blob/main/src/scss/dp/overrides/helpers/_margins.scss